go-controller/pkg/ovn/namespace.go

```diff
@@ -44,18 +44,22 @@ func (oc *Controller) addPodToNamespace(ns string, portInfo *lpInfo) error {
 	}
 	defer nsInfo.Unlock()
 
-	// If pod has already been added, nothing to do.
-	address := portInfo.ip.String()
-	if nsInfo.addressSet[address] != "" {
-		return nil
-	}
+	applied := false
+	for _, ip := range portInfo.ips {
+		// If pod has already been added, nothing to do.
+		address := ip.String()
+		if nsInfo.addressSet[address] != "" {
+			applied = true
+			continue
```

Comment:
If we assume that "if one IP is already added then all of them have been" then you should just `return nil` here like before. If we don't assume that, then you shouldn't skip the `podAddAllowMulticastPolicy` call when *any* IP is already added, you should only skip it if *all* of the IPs are already added.